src/wagtail_ai/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+
+from .prompts import Prompt as PromptDataClass
+
+
+class Prompt(models.Model):
+    label = models.CharField(max_length=255)
+    description = models.CharField(
+        max_length=255,
+        blank=True,
+        help_text="A description of the prompt to be displayed alongside it's label.",
+    )
+    prompt = models.TextField(
+        help_text=("The text prompt sent to the AI (e.g. ChatGPT) to generate content.")
+    )
+    method = models.CharField(
+        max_length=25,
+        choices=PromptDataClass.Method.choices,
+        default=PromptDataClass.Method.REPLACE,
```

Comment:
**Question:** Is there a reason why we need a default? Is this beneficial?

Reply:
Valid point, I thought it might be useful as a fallback incase people didn't specify one in the settings for `WAGTAIL_AI_PROMPTS`